Formula/zork.rb

```diff
@@ -0,0 +1,21 @@
+class Zork < Formula
+  desc "The Dugeon modified from FORTRAN to C"
+  homepage "https://github.com/devshane/zork"
+  url "https://github.com/devshane/zork/archive/a463c206cb0c714ba7483512a3a9edf543a7c2f5.tar.gz"
```

Comment:
Ack, request on https://github.com/devshane/zork/issues/3, happy to close this in the meantime if that's prefered on the Homebrew side.